### ISSUE TITLE
needs extra !is_null() check

### ISSUE THREAD

**Issue by peixotorms** [ivopetkov/html5-dom-document-php#24]
I got this error while doing some tests:

```
PHP Fatal error:  Uncaught Error: Call to a member function removeChild() on null in /.../libs/html5-dom-document-php-master/src/HTML5DOMDocument.php:155
Stack trace:
#0 /.../functions.php(748): IvoPetkov\HTML5DOMDocument->loadHTML('<meta data-html...', 8292)
```

Can be fixed, by editing HTML5DOMDocument.php line 155 to
`if(!is_null($headElement->parentNode)) { $headElement->parentNode->removeChild($headElement); }`

Please check if you need more !is_null() checks if you have some time.
 Thanks for this great library!

**Comment by ivopetkov** [comment]
Thanks. I'll fix it (hopefully soon, because I'm very busy these days).

**Comment by ivopetkov** [comment]
Can you please send me the code you pass to loadHTML(). It's kind of strange that the head element does not have a parent.

**Comment by peixotorms** [comment]
I don't have in anymore, but bare in mind that we can use php dom even if just for small fits and pieces of html, which may or may not be valid (ie: sections inside wordpress contents, which have some partially filtered html). Adding the checks everywhere where possible, will prevent these from happening.

Thanks

**Comment by ivopetkov** [comment]
Just released v2.0.1 when it's fixed.
